src/components/sheet/sheet-action-buttons/SheetActionButton.tsx

```diff
@@ -10,42 +10,70 @@ import styled from '@/styled-thing';
 import { position } from '@/styles';
 import ShadowStack from '@/react-native-shadow-stack';
 
-const addChartsStyling = isCharts =>
+type Props = {
+  borderRadius?: number;
+  color?: string;
+  disabled?: boolean;
+  elevation?: number;
+  emoji?: string;
+  forceShadows?: boolean;
+  icon?: string;
+  isCharts?: boolean;
+  isTransparent?: boolean;
+  label?: string;
+  lightShadows?: boolean;
+  marginBottom?: number;
+  nftShadows?: boolean;
+  onPress?: () => void;
+  scaleTo?: number;
+  size?: 'big';
+  testID?: string;
+  textColor?: string;
+  /**
+   * textSize accepts every key of fonts.size object which is a legacy dictionary of font sizes
```

Comment:
nit: more of a q but wonder if we should be moving to RDS when we do changes like this